src/styles/style.css

```diff
@@ -1,6 +1,29 @@
 @import './edit-form.css';
 
+:root {
+  --white: #fff;
+  --extra-light-grey: #f3f3f3;
+  --lighter-grey: #e3e4e4;
+  --grey: #363636;
+  --mineshaft: #323232;
+  --dark-grey: #303030;
+  --another-black: #222;
+  --darker-black: #1e1e1e;
+
+  --red: #d92b2b;
+  --red-on-dark: #ff5050;
+}
```

Comment:
importing these from `'./colours.css` did not work :question:

Reply:
Weird can't get it to work as well